src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
+    <form
       data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+      class="form"
+    >
```

Comment:
The form tag should have the correct `data-qa` attribute as specified in the task. Ensure that the big search form has `data-qa="big"` and the small search form has `data-qa="small"`.